packages/mui-base/src/Composite/Root/CompositeRoot.test.tsx

```diff
@@ -0,0 +1,266 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { act, createRenderer, fireEvent, flushMicrotasks } from '@mui/internal-test-utils';
+import { CompositeItem } from '../Item/CompositeItem';
+import { CompositeRoot } from './CompositeRoot';
+
+describe('Composite', () => {
+  const { render } = createRenderer();
+
+  describe('list', () => {
+    it('controlled mode', async () => {
+      function App() {
+        const [activeIndex, setActiveIndex] = React.useState(0);
+        return (
+          <CompositeRoot activeIndex={activeIndex} onActiveIndexChange={setActiveIndex}>
```

Comment:
`Composite` sets `data-active` internally, but we use `data-highlighted` for our components. We could remove the `data-active` attr internally in `Composite` and use the `highlighted: boolean` ownerState instead to apply `data-highlighted`

Reply:
Yeah, the `active` part of its `ownerState` should be removed, or changed to `highlighted`